live-examples/js-examples/date/date-toprimitive.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js" class="language-js">
```

Comment:
We don't need the `class="language-js"` attribute.

Also, having a new line after this opening `code` tag will make the example start with a blank line, and we'd prefer not to do that.

So this line should be like:

`<code id="static-js">const date = new Date('20 December 2019 14:48');`